setup/main.tf

```diff
@@ -42,21 +60,183 @@ resource "aws_security_group" "steampulse_rds_sg" {
   }
 
   tags = {
-    Name = "allow_tls"
+
+    Name = "steampulse_rds_sg"
   }
 }
 
-resource "aws_db_instance" "steampulse_database" {
-  allocated_storage      = 10
-  engine                 = "postgres"
-  instance_class         = "db.t3.micro"
-  identifier             = "steampulse-database"
-  db_name                = var.DATABASE_NAME
-  username               = var.DATABASE_USERNAME
-  password               = var.DATABASE_PASSWORD
-  db_subnet_group_name   = "public_subnet_group"
-  vpc_security_group_ids = [resource.aws_security_group.steampulse_rds_sg.id]
-  availability_zone      = "eu-west-2a"
-  skip_final_snapshot    = true
-  publicly_accessible    = true
-}
\ No newline at end of file
+resource "aws_security_group" "steampulse_pipeline_ecs_sg" {
+  name        = "steampulse_pipeline_ecs_sg"
+  description = "Allows traffic for the pipeline ECS"
+  vpc_id      = "vpc-0e0f897ec7ddc230d"
+
+  ingress {
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  egress {
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+
+
+  tags = {
+    Name = "steampulse_pipeline_ecs_sg"
+  }
+}
+
+
+#unfinished below
```

Comment:
The below resources that aren't commented out are working but I'm not 100% sure they have everything they need